Reference/Security/Security-settings/index.md

```diff
@@ -0,0 +1,49 @@
+# Umbraco Security Settings
+
+## Password settings
+
+The settings for Umbraco passwords are handled by default through the ASP.NET Membership Providers. There are two Membership Providers in the web.config file. One for Umbraco Members and one for Users.
+
+    <membership defaultProvider="UmbracoMembershipProvider" userIsOnlineTimeWindow="15">
+      <providers>
+        <clear />
+        <add name="UmbracoMembershipProvider" type="Umbraco.Web.Security.Providers.MembersMembershipProvider, Umbraco" minRequiredNonalphanumericCharacters="0" minRequiredPasswordLength="10" useLegacyEncoding="false" enablePasswordRetrieval="false" enablePasswordReset="false" requiresQuestionAndAnswer="false" defaultMemberTypeAlias="Member" passwordFormat="Hashed" allowManuallyChangingPassword="false" maxInvalidPasswordAttempts="50"/>
+        <add name="UsersMembershipProvider" type="Umbraco.Web.Security.Providers.UsersMembershipProvider, Umbraco"  enablePasswordRetrieval="false" enablePasswordReset="false" requiresQuestionAndAnswer="false" allowManuallyChangingPassword="false" maxInvalidPasswordAttempts="100"/>
+      </providers>
+    </membership>
+
+For both of these Membership Providers you can specify the following settings (if the attribute is not specified in the web.config the default value is used):
+
+- minRequiredNonalphanumericCharacters (default 0): The number of minimal non-alphanumeric characters that are required for a password
+- minRequiredPasswordLength (default 10): The minimal length of a password
+- useLegacyEncoding (default 'false'): Projects that have been upgraded from a pre Umbraco v7.6 version will have this setting set to 'true' as an older password encoding mechanism was used. This setting should not be used for new projects.
+- passwordFormat (default 'Hashed', possible options 'Clear', 'Encrypted', 'Hashed'): You can specify the way a password should be stored in the database. The options ‘clear’ and ‘encrypted’ should not be used.
+- enablePasswordReset (default 'false'): This setting is not used
+- enablePasswordRetrieval (default 'false'): This setting is not used
+- requiresQuestionAndAnswer (default 'false'): This setting is not used
+- allowManuallyChangingPassword (default 'false'): This setting is not used
+- maxInvalidPasswordAttempts (default 5): The number of attempts before an user or member is locked out
```

Comment:
suggested grammar improvement: although the letter u is a vowel, and therefore you expect to use 'an' instead of 'a', in the instance of User, the letter u, uses the Yu sound, and so 'a user' is preferred over 'an user'. Here and in other places in this PR, suggest switching to 'a user'